react-web/src/pages/auditing/reportUpload/ReportUpload.tsx

```diff
@@ -0,0 +1,317 @@
+          />
+
+          <TextArea
+            placeholder="Summary"
+            required={true}
+            minRows={4}
+            maxRows={4}
+            {...form.register("summary")}
+          />
+
+          <TextArea
+            placeholder="Disclaimer"
+            required={true}
+            minRows={4}
+            maxRows={4}
+            {...form.register("disclaimer")}
+          />
+
+          <TextArea
+            placeholder="Subject"
+            required={true}
+            maxRows={2}
+            {...form.register("subject")}
+          />
+
+          <div className="separator-label">Auditor Information</div>
+          <Input
+            label="Name"
+            type="text"
+            id="name"
+            required={true}
+            {...form.register("name")}
+          />
+
+          <Input
+            label="Website"
+            type="text"
+            id="website"
+            required={true}
+            {...form.register("website")}
+          />
+
+          <Input
+            label="Email"
+            type="text"
+            id="email"
+            required={true}
+            {...form.register("email")}
+          />
+
+          <Input
+            label="Logo"
+            type="text"
+            id="logo"
+            {...form.register("logo")}
+          />
+
+          <Input
+            label="Discord"
+            type="text"
+            id="discord"
+            {...form.register("discord")}
+          />
+
+          <Input
+            label="Twitter"
+            type="text"
+            id="twitter"
+            {...form.register("twitter")}
+          />
+
+          <div className="separator-label">Audit Report</div>
+          {/* <Upload
+            isMultiple={false}
+            highlightText="Upload a PDF"
+            uploadFiles={(file) => {
+              console.log("file", file);
+              setFiles(file);
+            }}
+            showPreview
+            tooltipText={`Please upload a ${SUPPORTED_FORMATS.join(",")} file within 5MB size`}
+            maxFileSize={FILE_SIZE}
+            acceptedTypes={SUPPORTED_FORMATS.join(",")}
+            uploadedFiles={files}
+            required={true}
+            className="bordered"
+            onClick={(files) => console.log(files)}
+            name="auditReport"
+            showDefaultError
+          /> */}
+          <TextArea
+            placeholder="Report URLs"
+            required={true}
+            maxRows={2}
+            tooltipText="Enter comma(,) separatated JSON/PDF URLs or IPFS link corresponding to the report"
+            {...form.register("reportURL")}
+          />
+
+          <div className="separator-label">DAPP Script</div>
+          <div className="relative">
+            <div className="absolute action-button addScript-btn">
+              <Button
+                displayStyle="primary-outline"
+                size="small"
+                buttonLabel="+ Add Script"
```

Comment:
Could this button be at the bottom, so that when we fill the script hashes, we don't have to scroll all the way up to click on it?

Reply:
Done --
![image](https://github.com/input-output-hk/dapps-certification/assets/11058223/c0b86c00-9d34-4b0c-bc08-715f42b6ad2f)